packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/OrderSummary.tsx

```diff
@@ -74,6 +100,31 @@ const OrderSummary: React.FC<Props> = ({
 
   const { days } = intervalToDuration({ end: lockTime, start: 0 })
 
+  const coinInterestRate = !!interestRate[outputCurrency] && interestRate[outputCurrency]
```

Comment:
Succinct, but hurts my brain to look at this. Mind leaving a short comment above it to explain what it's doing?

Reply:
Sure, thanks dylan